### PR TITLE
feat(ui5-tab-container): implement overflow-button slot

### LINKED ISSUE #2184: [Feature Request][A11Y][TabContainer] Need a parameter to custom the aria-label text for overflow menu button
**Dear colleagues,**
I am from SuccessFactors Web Experience Shanghai team.

**Feature request**
1. Need a parameter to custom the aria-label text for overflow menu button

FR  TabContainer

**Expected API Spec**

```
<ui5-tabcontainer class="full-width" show-overflow overflow-label="my customized overflow button label">
</ui5-tabcontainer>
```

### PR DESCRIPTION
**Pull request by fifoosid**
Fixes #2184

### CODE REVIEW

**Review by vladitasev — CHANGES_REQUESTED** [review]
The custom button is always there, no matter if an overflow menu is needed or not:
image

The normal button hides/appears properly

File: packages/main/src/TabContainer.js [hunk 1]
```diff
@@ -57,6 +57,19 @@ const metadata = {
...
+		 * Defines the buttons which is shown when there are overflowed items. If nothing is provided to this slot, the default button will be used.
+		 *
+		 * @type {HTMLElement[]}
+		 * @public
+		 * @slot
+		 * @since 1.0.0-rc.9
+		 */
+		overflowButton: {
+			type: HTMLElement,
+			invalidateParent: true,
```
[the remark this answers is not in the document]
Comment (fifoosid) [reply packages/main/src/TabContainer.js:71]: It was used in the initial implementation, but now it is not necessary

File: packages/main/src/TabContainer.js [hunk 2]
```diff
@@ -427,9 +440,14 @@ class TabContainer extends UI5Element {
 	}
 
 	async _onOverflowButtonClick(event) {
+		const button = this.overflowButton[0] || this.getDomRef().querySelector(".ui-tc__overflowButton");
 		this.responsivePopover = await this._respPopover();
 		this.updateStaticAreaItemContentDensity();
-		this.responsivePopover.open(this.getDomRef().querySelector(".ui-tc__overflowButton"));
+		if (this.responsivePopover.opened) {
```
[the remark this answers is not in the document]
Comment (fifoosid) [reply packages/main/src/TabContainer.js:446]: Yes, this is the intended behaviour which was not implemented until now.

File: packages/main/src/TabContainer.js [hunk 3]
```diff
@@ -57,6 +57,19 @@ const metadata = {
 			individualSlots: true,
 			listenFor: { include: ["*"] },
 		},
+
+		/**
+		 * Defines the buttons which is shown when there are overflowed items. If nothing is provided to this slot, the default button will be used.
```
[the remark this answers is not in the document]
Comment (fifoosid) [reply packages/main/src/TabContainer.js:62]: Done

**Review by vladitasev — APPROVED** [review]

**Pull request merged**